### PR TITLE
Strip whitespace in {% trans %} Jinja tags

### PR DESCRIPTION
**Pull request by pferreir**

### CODE REVIEW

File: indico/util/translations.py [hunk 1]
```diff
@@ -17,11 +17,26 @@
...
+
+
+# TODO: Remove this once there's proper support in upstream Jinja
+# https://github.com/pallets/jinja/pull/683
+def jinja2_babel_extract(fileobj, keywords, comment_tags, options):
+    """
+    Custom babel_extract for Jinja.
+
+    Hooks on to Jinja's babel_extract and handles
+    whitespace within `{% trans %}` tags.
```
Comment (ThiefMaster) [inline indico/util/translations.py]: double backticks

File: indico/util/translations.py [hunk 2]
```diff
@@ -17,11 +17,26 @@
...
     def wrapper(*args, **kwargs):
         return encode_if_unicode(fn(*args, **kwargs))
     return wrapper
+
+
+# TODO: Remove this once there's proper support in upstream Jinja
+# https://github.com/pallets/jinja/pull/683
+def jinja2_babel_extract(fileobj, keywords, comment_tags, options):
+    """
+    Custom babel_extract for Jinja.
```
Comment (ThiefMaster) [inline indico/util/translations.py]: fits in the previous line ;)

File: setup.py [hunk 3]
```diff
@@ -75,6 +75,9 @@
         'global_settings = indico_zodbimport.modules.global_settings:GlobalSettingsImporter',
         'badges_posters = indico_zodbimport.modules.badges_posters:BadgePosterImporter',
         'event_data = indico_zodbimport.modules.event_data:EventDataImporter',
+    ],
+    'babel.extractors': [
+        'jinja2_indico = indico.util.translations:jinja2_babel_extract'
```
Comment (ThiefMaster) [inline setup.py]: You can omit that and add this to `babel.cfg` instead in the `[extractors]` section:

    jinja2_indico = indico.util.translations:jinja2_babel_extract

File: indico/util/translations.py (hunk 2 above)
Comment (pferreir) [inline indico/util/translations.py]: I really don't like the asymmetry created by the lack of newline. But anyway, it's true that that's the style we mostly use.

File: setup.py (hunk 3 above)
Comment (pferreir) [inline setup.py]: You're right. Didn't recall that.

**Comment by ThiefMaster** [comment]
unittests are broken

**Comment by pferreir** [comment]
> unittests are broken

Apparently not by this... weird.

**Pull request merged**